### PR TITLE
updating change log for new release

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- updating changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
...
+
+### Added
+- Eightshift-forms as default allow block types.
+- New WP-CLI command to create custom WP-CLI command. Example: `wp boilerplate create_cli_command`.
+- New prop to outputCssVariables called $customSelector.
+- DI container caching on production, or staging.
+- Option to remove default paragraph block placeholder from the frontend if the content is empty.
+- Custom post type revisions as default.
+- Limit enqueue admin only to admin area and not block editor.
+- Option ot override default block class prefix in the blocks global settings manifest.
```
Comment (iobrado) [inline CHANGELOG.md:16]:
```suggestion
- Option to override default block class prefix in the blocks global settings manifest.
```

**Review by iobrado — APPROVED** [review]
Found just one typo 🙂

**Review by iobrado — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
 
+## [4.1.0] - 2022-02-02
+
+### Added
+- Eightshift-forms as default allow block types.
+- New WP-CLI command to create custom WP-CLI command. Example: `wp boilerplate create_cli_command`.
+- New prop to outputCssVariables called $customSelector.
+- DI container caching on production, or staging.
```
Comment (dadadavorin) [inline CHANGELOG.md:12]:
```suggestion
- DI container caching on production or staging.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
...
+- Option to remove default paragraph block placeholder from the frontend if the content is empty.
+- Custom post type revisions as default.
+- Limit enqueue admin only to admin area and not block editor.
```
Comment (dadadavorin) [inline CHANGELOG.md:15]:
```suggestion
- Limit enqueue admin only to admin area but not to the block editor.
```

**Review by dadadavorin — APPROVED** [review]
Looks good. Just few spelling suggestions

**Review by dadadavorin — APPROVED** [review]

**Review by iobrado — DISMISSED** [review]

**Review by simun-ivanac — DISMISSED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
...
+- New prop to outputCssVariables called $customSelector.
+- DI container caching on production or staging.
+- Option to remove default paragraph block placeholder from the frontend if the content is empty.
+- Custom post type revisions as default.
+- Limit enqueue admin only to admin area but not to the block editor.
+- Option to override default block class prefix in the blocks global settings manifest.
+- More tests.
+
+### Fixed
+- Changed directory separator from `\` to `DIRECTORY_SEPARATOR` constant to be able to work cross-platform.
```
Comment (mbmjertan) [inline CHANGELOG.md:20]: Previous separator is wrongly listed here, see PR #234
```suggestion
- Changed directory separator from `/` to `DIRECTORY_SEPARATOR` constant to be able to work cross-platform.
```

**Review by mbmjertan — CHANGES_REQUESTED** [review]

**Review by mbmjertan — DISMISSED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by simun-ivanac — DISMISSED** [review]

**Review by gabriel-glo — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
...
+- Option to remove default paragraph block placeholder from the frontend if the content is empty.
+- Custom post type revisions as default.
+- Limit enqueue admin only to admin area but not to the block editor.
+- Option to override default block class prefix in the blocks global settings manifest.
+- More tests.
+
+### Fixed
+- Changed directory separator from `/` to `DIRECTORY_SEPARATOR` constant to be able to work cross-platform.
+- Various fixes and improvements.
+
```
Comment (dingo-d) [inline CHANGELOG.md:22]: Can you add the missing releases? I've added them here, so just copy them from here: https://github.com/infinum/eightshift-libs/blob/3.1.0/CHANGELOG.md (just make sure to link them, I just saw that I didn't link them 🙈)

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
 
+## [4.1.0] - 2022-02-02
+
+### Added
+- Eightshift-forms as default allow block types.
+- New WP-CLI command to create custom WP-CLI command. Example: `wp boilerplate create_cli_command`.
+- New prop to outputCssVariables called $customSelector.
```
Comment (dingo-d) [inline CHANGELOG.md:11]:
```suggestion
- New attribute to `outputCssVariables` called `$customSelector`.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
...
+- DI container caching on production or staging.
+- Option to remove default paragraph block placeholder from the frontend if the content is empty.
+- Custom post type revisions as default.
+- Limit enqueue admin only to admin area but not to the block editor.
```
Comment (dingo-d) [inline CHANGELOG.md:15]: This should go under `Changed` section of the changelog

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
...
+- Option to override default block class prefix in the blocks global settings manifest.
+- More tests.
```
Comment (dingo-d) [inline CHANGELOG.md:17]: I don't think we need to mention we added more tests, that should be a given 😄

Comment (iruzevic) [reply CHANGELOG.md:17]: We sound smart :D

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left some minor comments, otherwise looks good to me. I pulled the develop branch locally for one project and I didn't see any breakage on a clients site, so I think we can tag this as 4.1.0 🙂

File: CHANGELOG.md (hunk 5 above)
Comment (iruzevic) [reply CHANGELOG.md:22]: Done

File: CHANGELOG.md (hunk 7 above)
Comment (iruzevic) [reply CHANGELOG.md:15]: done

File: CHANGELOG.md (hunk 8 above)
Comment (iruzevic) [reply CHANGELOG.md:17]: removed

**Pull request merged**